Update README.md -- improving docs (#39)
nautobot/nautobot-plugin-golden-config: README.md

A plugin for [Nautobot](https://github.com/nautobot/nautobot) that intends to provide context around golden configuration.

# Overview

> You may see the [Navigating Overview](./docs/navigating-golden.md) documentation for an overview of navigating through the different areas of this plugin. You may also see the [FAQ](./docs/FAQ.md) for commonly asked questions.

Reviewer: ```suggestion
You can see the [Navigating Overview](./docs/navigating-golden.md) documentation for an overview of navigating through the different areas of this plugin. You may also see the [FAQ](./docs/FAQ.md) for commonly asked questions.
```

Author: looks good

    * see [Navigating Intended](./docs/navigating-intended.md) for more information
* Source of Truth Aggregation - Is a GraphQL query per device with that creates a data structure used in the generation of configuration.
    * see [Navigating SoTAgg](./docs/navigating-sot-agg.md) for more information
* Configuration Compliance - Is a Nornir process to run comparison of the actual (via backups) and intended (via Jinja file creation) CLI configurations.
    * see [Navigating Compliance](./docs/navigating-compliance.md) for more information

The operator's of their own Nautobot instance are welcome to use any combination of these features. Though the appearance may seem like they are tightly 
coupled, this isn't actually the case. For example, one can obtain backup configurations from their current RANCID/Oxidized process and simply provide a Git Repo
of the location of the backup configurations, and the compliance process would work the same way. Also, another user may only want to generate configurations, but not want to use other features, which is perfectly fine to do so.

## Screenshots

> The plugin provides features and capabilities to integrate more workflows closely into the Nautobot ecosystem. The following screenshots are intended to provide a quick visual overview of some of these features.

Reviewer: ```suggestion
There are many features and capabilities the plugin provides into the Nautobot ecosystem. The following screenshots are intended to provide a quick visual overview of some of these features.
```

Author: > The plugin provides features and capabilities to integrate into the Nautobot ecosystem. The following screenshots are intended to provide a quick visual overview of some of these features.

if this sentence can be used instead it should make more sense. cheers
